### ISSUE TITLE
Extension is not working

### ISSUE THREAD

**Issue by nirmalsinghoo7** [vincaslt/vscode-highlight-matching-tag#115]
I have enabled all of the required settings. But the extension is not working.
![hight light 1](https://user-images.githubusercontent.com/12962402/90499958-dc2c6700-e167-11ea-8767-88b8f67934a3.png)

But it's not working. It just highlights whole tags.
It has to highlight the selected tag. Please see the screenshot. I have marked the selected div.
![hight light 2](https://user-images.githubusercontent.com/12962402/90500081-0716bb00-e168-11ea-9d61-3665c6fe8aa5.png)

**Comment by vincaslt** [comment]
Could you check if your console says anything? 
![image](https://user-images.githubusercontent.com/944727/90501592-44be1880-e155-11ea-975b-4f8b05d8c1fa.png)

It could also be that the highlighting that vscode puts on top in html files hides or overrides the one added by the extension - could you try creating a new file, and setting language as javascript, and pasting in:

```
<div attr></div>
```

then put cursor anywhere on attr (this will not trigger vscode highlighting) and see if it's working like this:
![image](https://user-images.githubusercontent.com/944727/90501839-a54d5580-e155-11ea-8098-30d4d374f2d1.png)

**Comment by nirmalsinghoo7** [comment]
Yes, It's showing two errors in the console. I have tried by uninstall and install again. but not working. I have tried in a new file. I have tried by .php and .html file extension, but not working
![console error](https://user-images.githubusercontent.com/12962402/90510683-09cddc00-e179-11ea-90f0-3b6d9d12de30.png)

**Comment by vincaslt** [comment]
Doesn't seem like the errors are from this extension. It could be that your settings JSON file is somehow invalid, could you try to resolve the errors, or disable other extensions - leaving only this one on?

**Comment by findyourneechie** [comment]
I am having the same issue. This has occurred since and update in august in vs code. All file extensions I've tested (html, css, js, php) do not underline and highlight anymore even with all selections checked on.

**Comment by vincaslt** [comment]
Can you try to change styling settings? The default ones might be hidden behind default highlighting in vscode.

**Comment by ruffin--** [comment]
I'm seeing something similar -- well, I'm also seeing nothing, I think. Does this look like the extension is working to you?

![simplest case test with cursor in attr](https://user-images.githubusercontent.com/5659878/128403625-27ac3849-c0f7-42bb-a49e-a3c98434e9b8.png)

I received no errors while creating that new file and placing my cursor in `attr`. 

Same deal with all other extensions disabled.

![disabled extensions](https://user-images.githubusercontent.com/5659878/128404091-8db98a03-a3c8-4d1d-81a5-75078955e4e7.png)

**Comment by vincaslt** [comment]
I'm pretty sure I can see yellow underline. It might be that default theme settings stopped working correctly for some reason or your theme reports it's color settings incorrectly (as dark theme instead of light). You can try changing color settings for this extension, e.g. to have black underline.

**Comment by ruffin--** [comment]
Yep, you're right. It's just understated. 

It is not working elsewhere because I'm using AngularJS templates, which are throwing errors (see below). I guess that's to be expected.

```
  ERR invalid syntax at line 146 col 21:

      <div class="pr-4"><strong>Date Created: </strong>{{Question.DateCreated | date:'medium'}}</div>
                      ^: Error: invalid syntax at line 146 col 21:

      <div class="pr-4"><strong>Date Created: </strong>{{Question.DateCreated | date:'medium'}}</div>
                      ^
    at Lexer._token (c:\Users\rbailey\.vscode\extensions\vincaslt.highlight-matching-tag-0.10.1\node_modules\moo\moo.js:533:13)
...
```

I wouldn't've necessarily expected that to kill tag matching, but I can see how it might!

Thanks, and apologies for the thread hijack.
